ocaml-lsp-server/src/ocamlformat.ml

```diff
@@ -0,0 +1,239 @@
+open Import
+
+module FileType = struct
```

Comment:
The convention in OCaml is to use underscores. It should be `File_type` in this case.

The reason why some identifiers are in fact in `PascalCase` is because they directly correspond to identifiers in the LSP spec.

Reply:
I didn't know about the convention, thanks! I'll fix it tomorrow.